### ISSUE TITLE
App is crashing on Windows 7

### ISSUE THREAD

**Issue by nuttyartist** [nuttyartist/notes#184]
A friend of mine tried to install the app, but get an error message that says that "MSVCP120.dll is missing".

**Comment by nuttyartist** [comment]
@theshadowx Maybe it's because our config on AppVeyour doesn't include that? Should we include that?

**Comment by theshadowx** [comment]
I think the installer you made doesn't include it, as windeployqt doesn't add it.
In the appveyor, I have manually added it the the folder from which the bunder will be generated : 
https://github.com/nuttyartist/notes/blob/dev/appveyor.yml#L63-L67

```
  - cd C:\ci\notes\bin
  - windeployqt .
  - if %compiler%==msvc2015 copy "C:\\Program Files (x86)\\Microsoft Visual Studio 14.0\\VC\\redist\\x86\\Microsoft.VC140.CRT\\msvcp140.dll" .
  - if %compiler%==msvc2015 copy "C:\\Program Files (x86)\\Microsoft Visual Studio 14.0\\VC\\redist\\x86\\Microsoft.VC140.CRT\\vcruntime140.dll" .
  - if %compiler%==msvc2015_64 copy "C:\\Program Files (x86)\\Microsoft Visual Studio 14.0\\VC\\redist\\x64\\Microsoft.VC140.CRT\\msvcp140.dll" .
  - if %compiler%==msvc2015_64 copy "C:\\Program Files (x86)\\Microsoft Visual Studio 14.0\\VC\\redist\\x64\\Microsoft.VC140.CRT\\vcruntime140.dll" .
  - iscc C:\ci\notes\packaging\windows\Notes_Inno_Script.iss
  - cd C:\ci\notes\installer

```

**Comment by nuttyartist** [comment]
But it states "MSVCP120.dll" on my friend's machine, not msvcp140.dll. And indeed, my fault, I probably forgot to add these necessary libraries.

**Comment by theshadowx** [comment]
yes because you used msvc 120 (2013) to compile the source code and the version used in appveyor is 140 (2015).

the dll follows the version of the compiler which you used

msvc 120 (2013) needs msvcp120.dll / vcruntime120.dll
msvc 140 (2015) needs msvcp140.dll / vcruntime140.dll

**Comment by theshadowx** [comment]
just to check : go to your appveyor Notes page and select

![selection_334](https://user-images.githubusercontent.com/976569/31854578-7336030c-b693-11e7-96ff-8afd9831be97.jpg)
 
You'll find in Artifacts tab: 

![appveyor_artifacts](https://user-images.githubusercontent.com/976569/31854550-3037d5d0-b693-11e7-8cd0-49a24db00e77.jpg)

Donwload the one with long name.

And tried it, in theory it won't tell you that dll is missing.

**Comment by nuttyartist** [comment]
Ok, I will check with my friend and update.

**Comment by nuttyartist** [comment]
Yeah, he said the link solved its problem. @theshadowx Are we still unable to create one Windows installer using AppVeyour?

**Comment by theshadowx** [comment]
I haven't found a solution yet.

**Comment by nuttyartist** [comment]
Ok, so I will create a new installer and upload it to the site. The real bummer here is that a new installer will probably lose all the popularity gained with the previous one, and thus, will be affected by Windows warnings. My fault on this.

**Comment by nuttyartist** [comment]
@theshadowx Is it good to use the AppVeyor's artifacts for the installer, rather for me to compile it every time on both 32bit and 64bit, and only packaging them together on 32bit VM?

**Comment by theshadowx** [comment]
I think it's better to use the package generated from appVeyor, as it's clean environment.

**Comment by nuttyartist** [comment]
Ok 👍

**Comment by guihkx** [comment]
This was fixed by #399, so let's close this.
